sdk/resourcemanager/Azure.ResourceManager/src/ArmClientOptions.cs

```diff
@@ -29,6 +29,8 @@ public sealed class ArmClientOptions : ClientOptions
         /// Sets the api version to use for a given resource type.
         /// To find which API Versions are available in your environment you can use the <see cref="ResourceProviderResource.Get"/> method
         /// for the provider namespace you are interested in.
+        /// To find which Resource Type to use you can refer to the corresponding Resource in the documentation or XML documentation comments of the API
```

Comment:
```suggestion
        /// To find which <see cref="ResourceType"/> to use you can refer to the corresponding Resource in the documentation or XML documentation comments of the API
```
what does this mean "corresponding Resource in the documentation"? Which documentation does this refer to?

Reply:
microsoft learning doc, or the xml doc comments shown in the intellisense